Develop

NaNNotABug opened the pull request:
[DEMO LINK] (https://NaNNotABug.github.io/layout_landing-page/).

NaNNotABug committed:
some message

NaNNotABug committed:
redesign files

NaNNotABug committed:
made scss blocks

NaNNotABug committed:
qa test1

NaNNotABug committed:
added mixins and media

NaNNotABug committed:
formatted code

NaNNotABug committed:
added recommended

NaNNotABug committed:
qa test1

NaNNotABug committed:
qa test2

NaNNotABug committed:
mixin???

NaNNotABug committed:
mixin???2

NaNNotABug committed:
img to width of page

NaNNotABug committed:
vse zihalo)

NaNNotABug committed:
add task solution

etojeDenys reviewed (changes requested):
1. place links on the same line as the logo
<img width="1420" height="677" alt="image" src="https://github.com/user-attachments/assets/af414e51-5e17-4120-9d74-10c003550311" />

2. this icon should be tel link
<img width="98" height="82" alt="image" src="https://github.com/user-attachments/assets/fc3c3031-2362-4694-9616-e918904f7c6f" />

3. on the desktop version, that icon should have a hover effect
<img width="295" height="109" alt="image" src="https://github.com/user-attachments/assets/64a912c3-9f22-4577-9c04-e159dd29b5b5" />

4. add favicon

5. decrease the scale a little so that the image does not overlap the other items
<img width="254" height="307" alt="image" src="https://github.com/user-attachments/assets/ebecd590-14f8-4b8b-9fea-c14c3b127dc9" />

6. this section looks different with the layout
your:
<img width="1431" height="412" alt="image" src="https://github.com/user-attachments/assets/12a80e5d-7cae-4e00-a0c7-a615ee637a7e" />

layout:
<img width="735" height="223" alt="image" src="https://github.com/user-attachments/assets/b5bd23c3-39ef-447f-b008-ef868eb3d017" />

7. these should be tel, mailto and address links

8. all inputs should be required

NaNNotABug committed:
add task solution 2

NaNNotABug committed:
added favicon

NaNNotABug commented:
1.fixed
2.fixed
3.it have hover(nothing changed)
4.added(on last deploy)
5.fixed(added this right before commit,didnt check it:( )
6.fixed
7.dont understand
8.fixed

NaNNotABug commented:
Шось зламалось на деплої,в мене періодично це стаєтся,перед останнім (адд фавікон) очистив парсел-кеш,не помогло(

FaiHamid reviewed (changes requested):
Feel free to ask for support in the FE__FX chat. Then after fixing it, redeploy and re-request a review.

NaNNotABug committed:
solve deploy problem

NaNNotABug committed:
i wish i fix deploy problem

NaNNotABug committed:
here we go again
